crossbeam-channel/benchmarks/chan.rs

```diff
@@ -9,7 +7,7 @@ const THREADS: usize = 4;
 
 fn new<T>(cap: Option<usize>) -> (chan::Sender<T>, chan::Receiver<T>) {
     match cap {
-        None => chan::async(),
+        None => chan::r#async(),
```

Comment:
Long term, should this name be changed not to clash with a keyword?

Reply:
The `chan` crate is deprecated so it probably won't be changed.